Editor/ChannelPackerTool.cs

```diff
@@ -41,7 +39,11 @@ public static void OpenEditorWindow()
         private void CreateGUI()
         {
             VisualElement root = rootVisualElement;
-            var visualTree = AssetDatabase.LoadAssetAtPath<VisualTreeAsset>("Assets/ChannelPackerTool/Resources/UI Documents/ChannelPackerToolWindow.uxml");
+            string[] paths = AssetDatabase.FindAssets("t:VisualTreeAsset ChannelPackerToolWindow");
```

Comment:
This just means you don't have to bring this library in at a specific path, instead it can be anywhere (for example I put it under `Assets/Plugins/Tools` which wouldn't work with the hardcoded path)